src/lib/Chip/Chip.svelte

```diff
@@ -1,10 +1,13 @@
 <script lang="ts">
-  import type { Variants } from './types'
+  type Variants = 'default' | 'info' |  'success' | 'warning' | 'danger';
 
   export let variant: Variants;
+
+  let klass = '';
+  export { klass as class };
 </script>
 
-<span class={`chip chip-${variant}`}>
+<span class={`chip chip-${variant} ${klass}`}>
```

Comment:
Maybe here instead of creating a specific prop for `class`, we might use `$$props.class`, like in https://svelte.dev/repl/6de3c939fcbf4c90a0309210c04e1c3a?version=3.17.2